clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -97,6 +97,8 @@ The improvements are...
 Improvements to clang-tidy
 --------------------------
 
+- Added `ctime` and `localtime` to clang-tidy.
```

Comment:
This belongs to `Changes in existing checks` section. Please also follow format for such entries.

Reply:
Sorry about that. I will move this.